python/rmm/rmm/librmm/cuda_stream_pool.pxd

```diff
@@ -1,12 +1,18 @@
-# SPDX-FileCopyrightText: Copyright (c) 2021-2024, NVIDIA CORPORATION.
+# SPDX-FileCopyrightText: Copyright (c) 2021-2025, NVIDIA CORPORATION.
 # SPDX-License-Identifier: Apache-2.0
 
 from rmm.librmm.cuda_stream_view cimport cuda_stream_view
+from libc.stdint cimport uint32_t
 
+cdef extern from "rmm/cuda_stream.hpp" namespace "rmm" nogil:
```

Comment:
We want to keep a 1-1 mapping of C++ headers to `.pxd` files. Can we put this in `python/rmm/rmm/librmm/cuda_stream.pxd` and then `cimport` it here?